CHANGELOG.md

```diff
@@ -451,4 +451,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [1.0.3]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.2...1.0.3
 [1.0.2]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.1...1.0.2
 [1.0.1]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.0...1.0.1
-[1.0.0]: https://github.com/rokwire/rokwire-building-blocks-api/releases/tag/1.0.0
```

Comment:
It could be an auto-merge-related thing, but the CHANGELOG entry for this task seems to be missing. Could you please add the entry and remove the currently highlighted change? Thanks.